src/components/exploreFurther/exploreFurther.tsx

```diff
@@ -72,10 +72,7 @@ const ExploreFurther = () => {
                 href={item.url}
                 target="_blank"
                 rel="noreferrer noopener"
-                sx={{
-                  textDecoration: "none !important",
-                  _hover: { textDecoration: "underline 1px dotted !important" },
-                }}
+                hasVisitedState={true}
```

Comment:
Nitpicky and not a necessary change but more of an FYI: passing the prop automatically sets it to true for boolean props. So in this can, `hasVisitedState={true}` is the same as `hasVisitedState`.